docker/planet/arm64-Dockerfile

```diff
@@ -13,13 +13,13 @@ RUN apk add --no-cache \
     jq \
       openssl
 
-COPY docker/planet/default.conf /etc/nginx/conf.d/
+COPY ./docker/planet/default.conf /etc/nginx/conf.d/
```

Comment:
there were working fine. any reason for changing them?

Reply:
suggestion from @dogi to be standard across the dockerfiles